runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -2,14 +2,22 @@
 
 import com.codahale.metrics.MetricRegistry;
 
+import com.google.common.collect.ImmutableList;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.collect.ImmutableList'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)